_dashboards/geojson-regionmaps.md

```diff
@@ -8,19 +8,25 @@ nav_order: 7
 
 OpenSearch Dashboards provides basic map tiles with a standard vector map that you can use to create your region map visualizations. You can configure the base map tiles using the Web Map Service (WMS) map server.
 
-You cannot configure a server to support user-defined vector map layers. However, you can configure your own GeoJSON file and upload it for this purpose. 
+You can't configure a server to support user-defined vector map layers. However, you can configure your own GeoJSON file and upload it for this purpose.
 {: .note}
 
 OpenSearch also has a standard set of GeoJSON files to provide a vector map with your regional maps.
 
 ## Custom vector maps with GeoJSON
 
-If you have a specific locale that is not provided by OpenSearch Dashboards vector maps, such as a US county or US ZIP Code, you can create your own custom vector map with a GeoJSON file. To use this feature, you have to install both the [Dashboards-Maps frontend plugin](https://github.com/opensearch-project/dashboards-maps) and the [OpenSearch Geospatial backend plugin](https://github.com/opensearch-project/geospatial).
+If you have a specific locale that is not provided by OpenSearch Dashboards vector maps, such as a US county or US ZIP Code, you can create your own custom vector map with a GeoJSON file. To create a custom region map you would define a geographic shape such as a polygon with multiple coordinates. To learn more about the various geographic shapes that support a custom region map location, see [Geoshape field type]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-shape/).
 
 GeoJSON format allows you to encode geographic data structures. To learn more about the GeoJSON specification, go to [geojson.org](https://geojson.org/).
 
 You can use [geojson.io](https://geojson.io/#map=2/20.0/0.0) to extract GeoJSON files.
-{: .tip}
+
+>   **PREREQUISITE**
+>   To use a custom vector map with GeoJSON, there are two required plugins that you need to install:
+>   * OpenSearch Dashboards Maps [`dashboards-maps`](https://github.com/opensearch-project/dashboards-maps) front-end plugin
+>   * OpenSearch [`geospatial`](https://github.com/opensearch-project/geospatial) backend plugin
```

Comment:
Should this be "back-end"? Similarly to "front-end" in the preceding bullet.

Reply:
frontend